_stbt/core.py

```diff
@@ -738,7 +738,7 @@ def _match_all(self, image, frame, match_parameters, region):
             # Remove this copy once we make numpy_from_sample *not* unmap the
             # sample when the contextmanager exits, but when no further
             # references are held to the numpy image.
-            npframe = numpy.copy(npframe)
+            npframe = npframe.copy()
```

Comment:
Hmmm...  or has it. Why are we still using `numpy_from_sample`? (Here and in `Display.push_sample`.)